src/deploy/deployHelper.js

```diff
@@ -0,0 +1,8 @@
+
+const getS3BucketPath = (branch) => {
+    return `branch-test/skyline/${branch.replace(/[^a-z0-9_\-]/gi, "_").toLowerCase()}`;
```

Comment:
for this to be in trues-shared, the repo needs to be an arg
branch-test is also quite specific, and context-dependent
do this instead, rename the function to getS3BranchFolderName, use the function syntax since it is easier to read:
```
function getS3BranchFolderName(branch) {
  return branch.replace(/[^a-z0-9_\-]/gi, "_").toLowerCase();
}

now if you actually need the full path, then you need the prefix to be passed in
basically it really needs to be the caller, since master or develop for Skyline is different than master or develop for C3, etc.